massa-bootstrap/src/lib.rs

```diff
@@ -37,7 +40,16 @@ async fn get_state_internal(
     bootstrap_public_key: &PublicKey,
     establisher: &mut Establisher,
     our_version: Version,
-) -> Result<(ExportProofOfStake, BootstrapableGraph, i64, BootstrapPeers), BootstrapError> {
+) -> Result<
+    (
+        ExportProofOfStake,
+        BootstrapableGraph,
+        i64,
```

Comment:
How I should guess what this `i64` is? Could we at least turn this tuple-struct into a named-struct, and maybe document a bit fields?

Reply:
(It's the compensation millis)

Reply:
thanks. Corrected